src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -112,6 +120,24 @@ export default function createPostGraphQLHttpRequestHandler (options) {
       return
     }
 
+    // Serve the JavaScript for GraphiQL on a namespaced path
+    if (parseUrl(req).pathname === '/_postgraphql/graphiql.js') {
+      res.statusCode = 200
+      res.setHeader('Content-Type', 'text/javascript')
+      res.end(await graphiqlJs)
+      return
+    }
+
+    // Setup an event stream so we can broadcast events to graphiql, etc.
+    if (parseUrl(req).pathname === '/_postgraphql/stream') {
+      if (req.headers.accept !== 'text/event-stream') {
+        res.end()
+        return
+      }
+      setupServerSentEvents(req, res, options)
```

Comment:
I think it looks fine. One way to make it more encapsulated is to perhaps make the function be a 1-to-1 representation of the event emitter. So maybe rename the function to `emitAsServerSentEvents(req, res, emitter)`? The either listen to all events from the event emitter or pass a map, something like: `new Map(['schemas:changed', { event: 'change', data: 'schema' }])` which maps event names to payloads?